USDA.NET.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27004.2008
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "USDA.NET", "src\USDA.NET\USDA.NET.csproj", "{5CBF44DC-E390-4065-AB11-98D6DD017E51}"
```

Comment:
Adding USDA.NET.csproj here is a requirement for merging (along with fixing the build). The following command now fails due to the refactor: `dotnet build tests/USDA.NET.Tests/USDA.NET.Tests.csproj`.

This however has brought to my attention that there is no build automation around this, so that's nice :)